docs/source/techniques/SANS-Toml-v1.rst

```diff
@@ -1322,37 +2137,139 @@ set centre a b c d [/MAIN] [/HAB]
         # Note for identical results the values will
         # only take a and b in the above example due to a bug
         # with the legacy user file parser
-        front_centre = {x=0.0842, y=-0.1965}
-        rear_centre = {x=0.0842, y=-0.1965}
+        front_centre = {x=0.31792, y=0.325498}
+        rear_centre = {x=0.32431, y=0.328547}
 
-set scales a b c d
-------------------
+SET SCALES a b c d e
+--------------------
+
+This command was used to specify the absolute intensity calibration scale
+factor (parameter a) to be applied to all intensity values at the end of
+the data reduction calculation.
+
+In the case of the LOQ instrument, it also allowed the relative scaling of
+the four high-angle detector banks (parameters b, c, d & e) to be accounted
+for (as a*b, a*c, a*d & a*e). For all other ISIS SANS instruments these
+four parameters should be set to unity.
+
+**Note: In 2020 it was discovered that due to a forever bug in the legacy
+User File command parser the parameters b, c, d & e have never been implemented
+in Mantid.** See this `issue <https://github.com/mantidproject/mantid/issues/27948>`_.
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
```

Comment:
So this bug is effectively here, where `b, c, d, and e` are completely left behind:
https://github.com/mantidproject/mantid/blob/cffb8050f43a522a2ff059a7834b0e4f59aa1075/scripts/SANS/sans/user_file/txt_parsers/ParsedDictConverter.py#L934

This means that it will fall back here https://github.com/mantidproject/mantid/blob/cffb8050f43a522a2ff059a7834b0e4f59aa1075/scripts/SANS/sans/algorithm_detail/scale_sans_workspace.py#L28
To always multiplying by `s` regardless of the workspace, or a factor of 100 if nothing was provided

Reply:
Ok. This is now clear in the Docs. And the https://github.com/mantidproject/mantid/issues/27948 will need to be addressed at a convenient point.